index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Nicolas | Portfolio</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<link href="https://fonts.googleapis.com/css2?family=Merriweather:ital@0;1&family=Work+Sans:wght@400;500;800&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/fa066ad49a.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+	<nav>
+		<div class="nav-link">
+			<a href="./indexfrench.html"> <img src="images/france.png" alt="french"></a>
+			<a href="#about_me">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact">Contact</a>
+
+		</div>
+		<div class="contact-button">
+			<a href="mailto:nicolas.herbreteau@gmail.com"><i class="fa-solid fa-paper-plane"></i></a>
+			<a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
+			<a href="#"><i class="fa-brands fa-github"></i></a>
+		</div>
+	</nav>
+	<header>
+		<img src="images/header.jpg" alt="coder">
+			<div class="intro_outerbox">
+				<div class="intro_container">
+					<a href="#about_me">&#8226;</a>
+					<a href="#portfolio">&#8226;</a>
+					<a href="#contact">&#8226;</a>
+				</div>
+				<div class="intro">
+					<h1>Welcome to Nico's Portfolio</h1>
+					<h3>Here you can browse some of the projects I have completed</h3>
```

Comment:
this should be a styled `<p>` because it does not serve the purpose of a heading on the page